### PR TITLE
Use replace_regex to replace all occurrences of "." with "/"

### PR DESCRIPTION
**Pull request by Thirumalai-Shaktivel**
Related: #1305

### CODE REVIEW

File: src/lpython/semantics/python_ast_to_asr.cpp [hunk 1]
```diff
@@ -3408,7 +3408,7 @@ class SymbolTableVisitor : public CommonVisitor<SymbolTableVisitor> {
                     // Check for the nested modules with "."
                     // Example: from x.y import z
                     if (mod_sym.find(".") != std::string::npos) {
-                        mod_sym.replace(mod_sym.find("."), 1, "/");
+                        mod_sym = replace(mod_sym, "[.]", "/");
```
Comment (czgdp1807) [inline src/lpython/semantics/python_ast_to_asr.cpp:3411]: Seems like this was hard coded before.

**Review by czgdp1807 — APPROVED** [review]
LGTM.

File: src/lpython/semantics/python_ast_to_asr.cpp (hunk 1 above)
Comment (Thirumalai-Shaktivel) [reply src/lpython/semantics/python_ast_to_asr.cpp:3411]: Previously, I thought `replace()` would replace every occrurances in the string. Now, I realized that it replaces only the 1st occurance or simple the index of the `fine()`.

**Comment by Thirumalai-Shaktivel** [comment]
Thanks for the approval. Let's merge this.

**Pull request merged**